templates/windows.tpl.yaml

```diff
@@ -25,10 +25,6 @@ metadata:
       /objects[0].spec.template.spec.volumes
       /objects[0].spec.template.spec.networks
 
```

Comment:
Please add the deprecation annotation like in the win2k12 deprecated template:
https://github.com/kubevirt/common-templates/blob/master/templates/win2k12r2-deprecated.tpl.yaml#L19

Reply:
and also please change the name of this file to `deprecated-windows.tpl.yaml` (make sure to change it in `generate-templates.yaml` as well)

Reply:
Done